Changelog.md

```diff
@@ -15,6 +15,7 @@ Compiler Features:
 
 Bugfixes:
  * General: Fix internal compiler error when requesting IR AST outputs for interfaces and abstract contracts.
+ * SMTChecker: Fix internal compiler error when analyzing overflowing expressions or bitwise negatation of unsigned types involving constants.
```

Comment:
```suggestion
 * SMTChecker: Fix internal compiler error when analyzing overflowing expressions or bitwise negation of unsigned types involving constants.
```

Reply:
Fixed. Sorry, you will have to re-approve.